core/main/src/broker/endpoint_broker.rs

```diff
@@ -821,12 +821,33 @@ pub trait EndpointBroker {
                             format!("{}_request", rpc_request.rpc.ctx.method),
                         );
                     }
+                    LogSignal::new(
```

Comment:
Do we also need a new LogSignal in the `if let Some(filter) = rpc_request
                        .rule
                        .transform
                        .get_transform_data(super::rules_engine::RuleTransformType::Request)` block, above?